awscli/customizations/wizard/ui/selectmenu.py

```diff
@@ -242,4 +260,15 @@ def move_up(event):
         def move_down(event):
             self._move_cursor(1)
 
+        @kb.add('f3', filter=Condition(self._prompt_has_details), is_global=True)
```

Comment:
So I was trying to keep the select menu classes fairly general without too much of the logic of how wizards work leak into it. To keep it relatively general, what are your thoughts on theses changes?

1. Switch the `details` parameter to be an `on_toggle` callback that can be provided and is called on any up or down key press. This callback would just accept a single value representing what the new selection is. We can then hoist this classes `_show_details()` method to the `WizardPromptSelectionAnswer` class. Another option I'm fine if we need to share this logic in multiple places is to just move the logic in the `_show_details()` method to a function in the`utils.py` module.

2. Add the keybinding for `f3` to the keybindings module. I'm also fine hoisting this to the `WizardPromptSelectionAnswer` if moving it to the keybindings module is not feasible.